_docs/setup/install-kubernetes.md

```diff
@@ -127,108 +127,42 @@ default namespace. They can be modified for deployment in a different namespace.
      ```
      This command will install Pilot, Mixer, Ingress-Controller, and Egress-Controller, and the Istio CA (Certificate Authority).
 
-1. *Optional:* Install addons for metric collection and/or request tracing as described in the following sections.
-
-### Enabling metrics collection
```

Comment:
that section should be green in some other doc in same PR no ?

Reply:
Nope. Its being removed. I spoke to @douglas-reid and he is going to rework this content in a different PR.

Reply:
IMO it's not a great experience to have it disappear completely even for a few days when it could be moved and linked

Reply:
I mean in a different task

Reply:
I assume the text is backed up in a branch...

Reply:
I added all the bloat again.